package.json

```diff
@@ -48,6 +48,7 @@
     "@types/cors": "^2.8.19",
     "@types/express": "^4.17.23",
     "@types/express-formidable": "^1.2.3",
+    "@types/jest": "^30.0.0",
```

Comment:
We are not using Jest 30 though, we are using Jest 29 for backend according to the lockfile